prime-angular-frontend/src/app/shared/components/site/summary-card/summary-card.component.html

```diff
@@ -25,6 +25,15 @@
       {{ actionButtonTitle }}
     </button>
   </mat-card-actions>
+  <mat-card-actions *ngIf="alternateButtonTitle">
```

Comment:
Feels like we don't need to set up multiple rows of actions and could show a single line of actions and provide attributes through the input binding actions array to apply appropriate colour - see below controller comment.